dev/io.openliberty.microprofile.telemetry.internal.common/bnd.bnd

```diff
@@ -74,7 +74,7 @@ Import-Package: \
 	io.opentelemetry.semconv:opentelemetry-semconv;version='${alphaSemconvOpenTelemetryVersion}';strategy=exact,\
 	com.ibm.ws.kernel.boot.core,\
 	com.ibm.ws.kernel.boot.common,\
-	io.openliberty.io.opentelemetry.2.0,\
+	io.openliberty.io.opentelemetry.2.1,\
```

Comment:
Not sure if this change is necessary.  Usually you compile to the lowest version that is used like you have with cdi 1.2.